.github/workflows/automated_release.yml

```diff
@@ -71,8 +71,15 @@ jobs:
           docker push
           docker.pkg.github.com/${IMAGE_NAME}:${{ env.RELEASE_VERSION }}
 
-      - name: Remove the oldest package
-        if: false
-        uses: actions/delete-package-versions@v1
+      - name: Remove old releases
+        uses: snok/container-retention-policy@v2
```

Comment:
for my understanding is this something developed in PSD or a standard (updated) bit of docker functionality?

Reply:
I believe the reason we do this is because we want to limit the amount of docker images we store on the github registry (to reduce storage usage).  The work came about from [this story](https://github.com/sanger/lighthouse/issues/475).